src/intersections.jl

```diff
@@ -115,3 +115,4 @@ include("intersections/planes.jl")
 include("intersections/boxes.jl")
 include("intersections/polygons.jl")
 include("intersections/domains.jl")
+include("intersections/sweepline.jl")
```

Comment:
Let's move this to our utils folder. It is a bit different than the other files in the intersections folder.